src/client/fitAddon.ts

```diff
@@ -0,0 +1,125 @@
+/**
```

Comment:
I'd suggest to add some paragraphs on top here to describe how and why this version is different